README.md

```diff
@@ -31,10 +27,22 @@ Example: `.eslintrc.js`:
 
 ```javascript
 module.exports = {
-  extends: ['@twostoryrobot/eslint-config/react']
+  extends: ['react-app', '@twostoryrobot/eslint-config/react']
```

Comment:
I know it's an example but this may throw an exception if copy-pasted for non-CRA apps (next.js or gatsby) if I understand right, since `react-app` is added by CRA. Maybe we make this just  `['@twostoryrobot/eslint-config/react']` and remove `Optionally` from the line below so that CRA has different instructions? Not sure what is best here. We could also just add a note explaining `react-app` might not be present on non-CRA apps.